[Security Solution] Advanced policy option for Endpoint 8.8

kevinlog opened the pull request:
## Summary

Addresses: https://github.com/elastic/kibana/issues/154476

Adds advanced policy option for Endpoint in 8.8.

Default:
<img width="1728" alt="image" src="https://user-images.githubusercontent.com/56395104/231883364-22d736b6-5555-4212-a05a-ceb52769ceba.png">

Set it to true and save:
![image](https://user-images.githubusercontent.com/56395104/230984178-e543e3af-2f49-48ad-af15-605b3a00336d.png)

<img width="1724" alt="image" src="https://user-images.githubusercontent.com/56395104/231883316-de19ff50-563f-4a11-ad22-d8ffd9c92312.png">

### Checklist

- [x] Any text added follows [EUI's writing guidelines](https://elastic.github.io/eui/#/guidelines/writing), uses sentence case text and includes [i18n support](https://github.com/elastic/kibana/blob/main/packages/kbn-i18n/README.md)

elasticmachine commented:
Pinging @elastic/security-defend-workflows (Team:Defend Workflows)

kevinlog commented:
@elasticmachine merge upstream

nicholasberlin reviewed (approved):
LGTM, though I wonder about the markdown that I added to the text blurb. The ` could be removed I suppose, or replaced with double quotes. Also, removing the the ` around fchdir in either case would be good.

kevinlog commented:
@elasticmachine merge upstream

kevinlog commented:
@nicholasberlin thanks, I removed the ` and updated the screenshots commit: https://github.com/elastic/kibana/pull/154684/commits/44ce8a19dadd2c2616ec3975255880f1a81d29dc
